projects/cdk/components/alert-host/alert-host.component.ts

```diff
@@ -23,10 +27,23 @@ import {Observable} from 'rxjs';
     encapsulation: ViewEncapsulation.None,
 })
 export class TuiAlertHostComponent<T extends TuiDialog<unknown, unknown>> {
+    public alerts: readonly (readonly T[])[] = [];
```

Comment:
I think there's a lint rule for such types to be written as `ReadonlyArray<readonly T[]>`. We only use `readonly` modifier for simple array types, otherwise it becomes hard to read.